terraform/ec2/amis.tf

```diff
@@ -398,7 +398,7 @@ data "aws_ami" "selected" {
 
   filter {
     name   = "architecture"
-    values = [var.amis[var.testing_ami]["arch"]]
```

Comment:
Are there any other places that need this change?

Reply:
no